Add support for RSMods `Disable Song Previews` Mod

ffio1 opened the pull request:
Rocksniffer looks for the `_Preview` suffix on the song previews to get the SongKey for the song. To disable song previews, RSMods changes this Wwise event from `Play_{SongKey}_Preview` to `Play_{SongKey}_Invalid` so Wwise doesn't know what event to play.

This will allow Rocksniffer to work while the `Disable Song Previews` mod is enabled.

ffio1 committed:
Add support for RSMods `Disable Song Previews` Mod

Rocksniffer looks for the "_Preview" suffix on the song previews to get the SongKey for the song. To disable song previews, RSMods changes this Wwise event to "Play_{SongKey}_Invalid". This will allow Rocksniffer to work while the mod is enabled.

kokolihapihvi commented on RSHelpers/RSMemoryReader.cs:
```diff
@@ -50,10 +50,10 @@ public RSMemoryReadout DoReadout()
             {
-                //Verify Play_ prefix and _Preview suffix
-                if (preview_name.StartsWith("Play_") && preview_name.EndsWith("_Preview"))
+                //Verify Play_ prefix and _Preview or _Invalid suffix
```
I would like a comment to briefly explain that _Invalid is a RSMod replacement to avoid possible future head scratching.

ffio1 committed:
Added comment regarding "_Invalid" suffix

ffio1 commented:
I submitted a new commit that has two comments explaining what the "_Invalid" suffix means, and why we need to look for it.

ffio1 committed:
Edit comment

After re-reading a couple times, I felt that this slight adjustment was needed to make the comment more clear.

kokolihapihvi commented:
Thank you!
